gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/JobStatusRetrieverTest.java

```diff
@@ -218,6 +222,40 @@ public void testGetLatestExecutionIdsForFlow() throws Exception {
     Assert.assertEquals(this.jobStatusRetriever.getLatestExecutionIdForFlow(FLOW_NAME, FLOW_GROUP), -1L);
   }
 
+  @Test (dependsOnMethods = "testGetLatestExecutionIdsForFlow")
+  public void testGetJobStatusesForFlowExecutionWithDagManager() throws Exception {
```

Comment:
do we expect the same result whether or not the DAG Manager is enabled?  if so, should we initialize the status once in common and then have each JobStatusRetriever analyze the same thing (to get the same result)?